easy-setup.sh

```diff
@@ -0,0 +1,80 @@
+OS_BASE=$"(uname)"
```

Comment:
This must be changed. The double quotes must be changed to `

or the script will fail as the variable OS_BASE will not have the correct value.

Reply:
```suggestion
OS_BASE=$`(uname)`
```